--- github.com/ipfs/kubo/vendor/github.com/libp2p/go-openssl/build.go	2022-08-30 05:56:01.000000000 -0500
+++ github.com/ipfs/kubo/vendor/github.com/libp2p/go-openssl/build.go	2022-09-20 18:01:06.606953798 -0500
@@ -18,7 +18,7 @@

 // #cgo linux windows freebsd openbsd solaris pkg-config: libssl libcrypto
 // #cgo linux freebsd openbsd solaris CFLAGS: -Wno-deprecated-declarations
-// #cgo darwin CFLAGS: -I/usr/local/opt/openssl@1.1/include -I/usr/local/opt/openssl/include -Wno-deprecated-declarations
-// #cgo darwin LDFLAGS: -L/usr/local/opt/openssl@1.1/lib -L/usr/local/opt/openssl/lib -lssl -lcrypto
+// #cgo darwin CFLAGS: -Wno-deprecated-declarations
+// #cgo darwin LDFLAGS: -lssl -lcrypto
 // #cgo windows CFLAGS: -DWIN32_LEAN_AND_MEAN
 import "C"
